src/include/txn.i

```diff
@@ -867,17 +927,20 @@ __wt_txn_read(WT_SESSION_IMPL *session, WT_CURSOR_BTREE *cbt, WT_ITEM *key, uint
         cbt->upd_value->buf.size = vpack->size;
     }
 
+    visible_type = __wt_txn_tw_stop_visible_type(session, &tw);
```

Comment:
Before the change, we doesn't check prepare conflict for the on disk value, which is incorrect I think.

Reply:
Looks like it's not an issue according to @kommiharibabu 's comment in WT-5849.

Reply:
Revert 42ef48b